apps/music/js/db.js

```diff
@@ -122,7 +122,13 @@ function initDB() {
     App.showCurrentView(function() {
       reparsingMetadata = false;
       // Hide the  spinner once we've displayed the initial screen
-      document.getElementById('spinner-overlay').classList.add('hidden');
+      // Setting the value stops the animation and works around bug 962594.
+      // The setTimeout is a workaround for Bug 1166500.
+      document.getElementById('spinner').value = 0;
```

Comment:
You use `.value = 0` here, but `.setAttribute('value', 0)` above. I'm sure they're functionally the same, but maybe we should prefer a single form in all cases...